assets/admin/editor-wizard/steps/lesson-patterns-step.js

```diff
@@ -1,43 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
-import PatternsList from '../patterns-list';
+import PatternsStep from './patterns-step';
 
 /**
- * Final step in lesson creation wizard choosing the actual lesson pattern to use.
+ * Lesson patterns step.
  *
- * @param {Object}   props
- * @param {Object}   props.data
- * @param {Function} props.setData
- * @param {Function} props.onCompletion
+ * @param {Object} props Component props.
  */
-/* eslint-disable no-unused-vars */
-const LessonPatternsStep = ( {
-	data: wizardData,
-	setData: setWizardData,
-	onCompletion,
-} ) => {
-	// TODO Implement this.
-
-	// We can call `onCompletion` to complete the wizard after setting the correct pattern with `setData`.
-	// We could replace `onCompletion` with the `goToNextStep` callback with a similar effect.
-
-	return (
-		<div className="sensei-editor-wizard-modal__content">
-			<h1>Lesson Patterns Step</h1>
-			<div>PENDING TO IMPLEMENT</div>
-			<PatternsList />
-		</div>
-	);
-};
+const LessonPatternsStep = ( props ) => (
+	<PatternsStep title={ __( 'Lesson Type', 'sensei-lms' ) } { ...props } />
```

Comment:
I guess we will add replacements here for the title (if finally are needed).

Reply:
Sorry! Not sure if I got your point here. Could you explain more?

Reply:
Ah, reading again, I think it's the same you meant with this following sentence, right?

> Lesson step will need to be implemented after https://github.com/Automattic/sensei/pull/5194 is integrated, right?

Reply:
That's it! :D